src/js/jsx/PanelSet.jsx

```diff
@@ -145,14 +142,11 @@ define(function (require, exports, module) {
         },
 
         componentDidUpdate: function (prevProps, prevState) {
-            // FIXME: Remove this once we ship with libraries always enabled
-            if (!prevState.librariesEnabled && this.state.librariesEnabled) {
-                this.getFlux().actions.libraries.beforeStartup()
-                    .bind(this)
-                    .then(function () {
-                        this.getFlux().actions.libraries.afterStartup();
-                    });
-            }
+            this.getFlux().actions.libraries.beforeStartup()
```

Comment:
These are no longer necessary. The reason I've added them in the first place was to make sure connection was initialized if we swithced from disabled to enabled state. BeforeStartup and afterStartup will be automatically called by FluxController on startup